Beitrags_und_Mahnordnung.md

```diff
@@ -1,20 +1,20 @@
 #Beitrags- und Mahnordnung
 *des Netz39 e.V. - Hackerspace Magdeburg*
 
-in der Fassung vom 06.05.2012
-
-Im Zuge der Gründungsversammlung am 06.05.2012 wurde die Beitrags- und Mahnordnung in der vorliegenden Fassung von der Gründungsversammlung beschlossen
+in der Fassung vom 14.10.2015.
 
 ###§1 Beitragsordnung
-1. Der Mitgliedsbeitrag ist jeweils zum Beginn eines Quartals für das gesamte Quartal im Voraus zu entrichten oder kann per Lastschriftverfahren eingezogen werden.
+1. Der Mitgliedsbeitrag ist jeweils zum Beginn eines Kalenderquartals für das gesamte Quartal im Voraus zu entrichten oder kann per Lastschriftverfahren eingezogen werden.
+
+2. Im Falle eines Beitritts innerhalb eines Quartals, wird eine Ermäßigung des Quartalsbeitrages gewährt. Diese beträgt für jeden bereits abgelaufenen Kalendermonat des aktuellen Quartals ein Drittel des Quartalsbeitrages.
```

Comment:
Hier ist die Änderung, welche sich von Renés Vorschlag ( https://github.com/netz39/Ordnungen/pull/1/files ) unterscheidet. Statt einer freiwilligen Spende im Beitrittsquartal wird für jeden beendeten Monat eine Ermäßigung in Höhe des alten Monatsbeitrages gewährt.

Bsp. 0:
* Beitritt am 12. April 1961
* Zu Zahlen: 100 EUR = 10 EUR (Beitrittsgebühr) + 90 EUR (Quartalsbeitrag)

Bsp. 1: 
* Beitritt am 24. Mai 1949
* Zu Zahlen: 70 EUR = 10 EUR (Beitrittsgebühr) + 90 EUR (Quartalsbeitrag) - 30 EUR (1/3 Quartalsbeitrag für abgelaufenen April)

Bsp. 2:
* Beitritt am 14. Juni 1985
* Zu Zahlen: 40 EUR = 10 EUR (Beitrittsgebühr) + 90 EUR (Quartalsbeitrag) - 2 * 30 EUR (2* 1/3 Quartalsbeitrag für abgelaufenen April und Mai)